finance_manager/src/main/java/com/exadel/finance/manager/user/service/api/UserService.java

```diff
@@ -0,0 +1,17 @@
+package com.exadel.finance.manager.user.service.api;
+
+import com.exadel.finance.manager.user.model.entity.User;
+import com.exadel.finance.manager.user.model.entity.UserModel;
+
+public interface UserService {
+
+    User createUser(UserModel user);
+
+    User readUser();
```

Comment:
how would readUser(), deleteUser() and getLoggedInUser() know which exactly User we are referring? Should have identifier